plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2036,61 @@ def coupling_parameter(
         )
 
     return coulomb_energy / kinetic_energy
+
+
+@validate_quantities(
+    n_b={"can_be_negative": False},
+    T_b={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+)
+class RelaxationRates:
+    def __init__(
```

Comment:
Whoa, if this actually works then @rocco8773 is an even greater wizard than I thought! AFAIK you have to decorate `__init__` instead of the main class declaration (move the decorator one line down 😉)
```suggestion
class RelaxationRates:
	@validate_quantities(
	    n_b={"can_be_negative": False},
	    T_b={"can_be_negative": False, "equivalencies": u.temperature_energy()},
	)
    def __init__(
```

Reply:
🤦 This is what happens when I try to program too early in the morning

Reply:
Hey, some of my best code was written before getting fully dressed in the morning! Some of my worst, too, as it happens. 😅